bounties_api/std_bounties/serializers.py

```diff
@@ -34,6 +34,10 @@ class Meta:
         model = Category
         fields = '__all__'
 
+class LanguageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Language
+        fields = '__all__'
```

Comment:
For now the endpoint only really needs to return the normalized and display name. I like that we have it stored in the DB, but for now we can just exclude native name here.

Reply:
👍